charts/pomerium/templates/tls-secrets.yaml

```diff
@@ -159,32 +159,33 @@ data:
   tls.crt: {{ $kp.Cert | b64enc }}
   tls.key: {{ $kp.Key | b64enc }}
 {{-   else if not .Values.config.generateTLS }}
-{{-     if and (not .Values.config.existingCASecret) .Values.config.ca }}
+{{-     if and (not .Values.config.existingCASecret) .Values.config.ca.cert .Values.config.ca.key }}
 {{ template "pomerium.ca.tlsSecretObject" . }}
-  {{ template "pomerium.caSecret.certName" . }}: {{ .Values.config.ca | b64enc }}
+  ca.crt: {{ .Values.config.ca.cert | b64enc }}
+  ca.key: {{ .Values.config.ca.key | b64enc }}
 {{-     end }}
 {{-     if and (not .Values.authenticate.existingTLSSecret) .Values.authenticate.tls.cert .Values.authenticate.tls.key }}
 ---
 {{ template "pomerium.authenticate.tlsSecretObject" . }}
-  {{ template "pomerium.authenticate.tlsSecret.certName" . }}: {{ .Values.authenticate.tls.cert | b64enc }}
-  {{ template "pomerium.authenticate.tlsSecret.keyName" . }}: {{ .Values.authenticate.tls.key | b64enc }}
+  tls.crt: {{ .Values.authenticate.tls.cert | b64enc }}
+  tls.key: {{ .Values.authenticate.tls.key | b64enc }}
```

Comment:
@tgomas `pomerium.authenticate.tlsSecret.keyName` and others shouldn't have been removed from templates then?

Reply:
They were not in templates, hence the bugfix

Reply:
ok, thx for the explanation :)